neighborhood_data/generate_neighborhood_json.py

```diff
@@ -121,7 +121,10 @@
                     try:
                         properties[field] = float(val)
                     except ValueError as ex:
-                        if field == 'crime_percentile':
+                        if (
+                            field == "crime_percentile"
+                            or field == "education_percentile"
+                        ):
```

Comment:
This is the script run locally to generate the new neighborhoods.json from an updated csv. The `education_percentile` field is non-nullable on the `Neighborhood` model but there are some neighborhoods where we don't have this value available,[ see this slack discussion.](https://element84.slack.com/archives/C04U48AN9N3/p1766012832815599) I found in practice the best fix was to handle it as we do `crime_percentile`. The frontend will use defaults where needed if `education_percentile` is false-y and this allows the script to pass even if there are non-float values in the csv